### PR TITLE
Adding setup templates to the ML rules

### PR DESCRIPTION
**Pull request by sodhikirti07**
## Issues
- https://github.com/elastic/ia-trade-team/issues/324

## Summary
This PR adds related setup template to the ML rules.

Setup template: [Setup Templates (ML rules)](https://docs.google.com/document/d/1EE0KpFf008AO_ee57ZUwuE_Njeud1ZESO3RvZ-pUGvQ/edit)

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/integrations/aws/ml_cloudtrail_error_message_spike.toml [hunk 1]
```diff
@@ -23,6 +23,27 @@ interval = "15m"
 license = "Elastic License v2"
 machine_learning_job_id = "high_distinct_count_error_message"
 name = "Spike in AWS Error Messages"
+setup = """## Setup
```
Comment (shashank-elastic) [inline rules/integrations/aws/ml_cloudtrail_error_message_spike.toml:26]: The note field, also has existing Setup information and the Unit tests are failing as there are 2 setup informations in Rule.  Please adjust these and remove the already existing setup information.

Comment (shashank-elastic) [reply rules/integrations/aws/ml_cloudtrail_error_message_spike.toml:26]: Please check this across all rules, and also update updated_date filed to reflect rule modification date.

**Review by shashank-elastic — CHANGES_REQUESTED** [review]

**Review by shashank-elastic — APPROVED** [review]
Templates reviewed here https://github.com/elastic/ia-trade-team/issues/324#issuecomment-2165399171. 
This is implementation PR.

**Review by jmcarlock — APPROVED** [review]

**Review by susan-shu-c — APPROVED** [review]

**Review by approksiu — APPROVED** [review]
Apart from rules with 2 setup fields, LGTM.

**Comment by sodhikirti07** [comment]
> Apart from rules with 2 setup fields, LGTM.

@approksiu I've removed the second set of setup instructions from the rules.

**Pull request merged**